src/browser/index.jsx

```diff
@@ -395,6 +401,12 @@ var MattermostView = React.createClass({
       }
     });
 
+    webview.addEventListener('page-title-updated', function(event) {
```

Comment:
Well ```index``` is only known outside of ```MattermostView```so I have no idea to get that working unfortunatly. And I can't set the prop from outside of ```MattermostView```